lib/markdown-it/markdown-it-radio-button.js

```diff
@@ -18,7 +18,7 @@ module.exports = function(md, options) {
         if (group) {
           group = group[1];
         } else {
-          group = Math.random().toString(36).substr(2, 5); // generate a new group id
+          group = md5(tokens[i-5].content + tokens[i-4].content + tokens[i].content).substr(2, 5); // generate a deterministic group id
```

Comment:
.substr(2, 5) is only used for getting a short random id. We could use a longer one if we go with md5 way (or just getting rid of this).

Reply:
Thanks for the `crypto` suggestion!

Oddly, sometimes it doesn't give the same value as the `md5` function, but it's fine for our use.

I think it's nice to keep it short. I doubt there would be a collision anyway. What's your thought?

Reply:
Let's keep it for now then. It should work fine for now and we could always change it later :)